libs/vtu/type.ts

```
@@ -1,17 +1,13 @@
+  type: VirtualDOMType;
   props: Record<string, unknown> | null;
-  children: (VirtualDOM | VirtualNode)[];
+  children: (VirtualNode | VirtualDOMType)[];
 };
 
-export type VirtualNodeType =
-  | HTMLElementTagNameMap
-  | string
-  | number
-  | Function;
+export type VirtualDOMType = HTMLElementTagNameMap | string | number | Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)